manifests/init.pp

```diff
@@ -81,6 +84,22 @@
   #Merged values provides by user and default values
   $_real_padoptions = merge($default_padoptions, $padoptions)
 
+  #Install choosing plugins
+  $plugins_list.each |$_pname, $_penable| {
+  if $_penable == true {
```

Comment:
can you turn this into a `case` block, instead of `if`?

Reply:
I'll try.

Reply:
Can you give me some details to create the `case` ? 
I want to recover values of `plugins_list` to match cases (true, false, undef) and use `keys` to have the plugin name.

Reply:
Fixed.